### ISSUE TITLE
a11y scrolling broken on Android

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#12586]
Not sure when this happened, but when the a11y focus reaches the last partially visible element of a scrollable list, that element is no longer brought into view. Also, when the focus moves past the last visible element, the list is no longer scrolled.

**Comment by goderbauer** [comment]
Looks like something on Android broke as this problem is not limited to Flutter apps. a11y scrolling also doesn't work anymore in native android apps (e.g. settings).

**Comment by goderbauer** [comment]
Interesting: Nexus 6P with Android 7.1.2 and TalkBack 5.2.1 is broken.

My other phone (Moto 4G with Android 7.0 and TalkBack 5.2.1) is unaffected.

**Comment by goderbauer** [comment]
Re-installing the TalkBack apk fixed this.
